lms/static/coffee/spec/instructor_dashboard/send_email_spec.coffee

```diff
@@ -0,0 +1,82 @@
+describe "Bulk Email Queueing", ->
```

Comment:
Yeah, I see not wanting to waste time rewriting something that works for the moment, but I'd argue that adding new code in a language we've essentially deprecated doesn't increase overall velocity - someone still has to maintain this, and the cost of maintaining CS is higher than JS due to its relative rarity. It's tech debt from the moment it's merged.